### PR TITLE
Remove trailing whitespace

### PR DESCRIPTION
**Pull request by masterleinad**
This is for all `CMakeLists.txt`, `*.cc`, `*.h`, `*.html` and `*.dox` files in `include`, `source`, `doc`, `contrib` and `examples`.

### CODE REVIEW

**Comment by masterleinad** [comment]
> What command did you use to do this check?

I used 
```
find source include/ examples/ contrib/ doc -type f \( -name "CMakeLists.txt" -o -name "*.cc" -o -name "*.h" -o -name "*.html" -o -name "*.dox" \) -exec sed -i "s/\s\+$//g" {} \;
```

**Comment by masterleinad** [comment]
I found some more after rebasing to `master` and force-pushed.

**Comment by bangerth** [comment]
Independent from this PR, do you want to make that script part of `contrib/utilities/indent`?

**Comment by tjhei** [comment]
> Independent from this PR, do you want to make that script part of `contrib/utilities/indent`?

:+1: That's what I was thinking about as well.

**Comment by masterleinad** [comment]
Alright, I added a script (and forced-pushed again due to more trailing whitespaces after rebasing).

File: contrib/utilities/indent_common.sh [hunk 1]
```diff
@@ -162,6 +162,16 @@ format_file()
 }
 export -f format_file
 
+#
+# Remove trailing whitespace
+#
+
+remove_trailing_whitespace()
+{
+  sed -i 's/\s\+$//g' $1
```
Comment (drwells) [inline contrib/utilities/indent_common.sh]: oops: `sed -i` is not portable!

**Pull request merged**